Adiciona conteúdo da seção Passo a Passo (#23)
pugpe/meetup-cookbook: languages/portuguese/arquivos/PASSO-A-PASSO.md

# Passo a passo inicial
> Aqui está descrito o fluxo principal para criação de um encontro do PUG-PE

Se você está aqui e quer organizar um encontro do PUG-PE, mas não sabe por onde começar, relaxa e se liga nesse checklist :stuck_out_tongue_closed_eyes: :star:

## Antes de organizar o evento
*A maioria dos pontos devem ser preenchidos*

- [ ] **Ter vontade de organizar um encontro**

> - [ ] **Ter amiguinhos com vontade de ajudar você nessa jornada (não precisamos fazer tudo sozinhos, não é mesmo?)**

Reviewer: Que que tu acha de trocar amiguinhos por colegas? Não que seja um grande problema deixar "amiguinhos" 
```suggestion
- [ ] **Ter colegas com vontade de ajudar você nessa jornada (não precisamos fazer tudo sozinhos, não é mesmo?)**
```

  
  Caso você queira companhia nessa jornada, esse ponto pode ser levado em consideração.
  
  Se você é estudante em alguma Universidade/Faculdade/Escola e está querendo organizar o encontro onde você estuda, recomendamos que você converse com seus colegas. Provavelmente, terá alguém disposto a te ajudar em alguma coisa :smile: 
  

>   Se, infelizmente, isso não ocorrer relaxa. Temos pessoas muito legais e dispostas na organização do PUG-PE que podem te ajudar nessa nova empreitada. 

Reviewer: ```suggestion
  Se, infelizmente, isso não ocorrer, relaxa. Temos pessoas muito legais e dispostas na organização do PUG-PE que podem te ajudar nessa nova empreitada. 
```

  
- [ ] **Ter disponibilidade (ou pessoas disponíveis) para organizar o evento**
  

>   Fazer divulgação, gerenciar submissão de palestras, montar grade de horario, organizar o evento no dia, resolver certificados, isso demanda tempo e não deve ser subestimado.

Reviewer: Não necessariamente num evento do PUG tem que ter certificado, né? Tu não acha que seria melhor sinalizar bem isso, pra que quem quiser organizar não ache que necessariamente tenha que fazer isso?

>   Esse ponto está fortemente relacionado com o anterior, se você acha que sua disponibilidade não é suficiente para fazer tudo isso, recomendamos que você tenha mais alguém com você (mais de duas pessoas é desnecessário). 

Reviewer: Dizer que algo é desnecessário é meio forte, principalmente sem saber quem são as pessoas que tão organizando e como elas se organizam, ai sugeri isso aqui, mas se tu achar melhor escrever de outra forma :man_shrugging:
```suggestion
  Esse ponto está fortemente relacionado com o anterior, se você acha que sua disponibilidade não é suficiente para fazer tudo isso, recomendamos que você tenha mais alguém com você (mais de duas pessoas pode ser desnecessário). 
```

  

---

## Pondo as ideias no lugar
*Todos os pontos devem ser preenchidos*

> - [ ] **Quero fazer um evento mais introdutorio ou avançado?**

Reviewer: Não sei se concordo muito com essa, acho que é possível e acho que desejável fazer um evento diverso que possa ser atrativo tanto pro público mais experiente, tanto pro público iniciante. Que que tu acha?

  
  Esse ponto é importante, pois a partir dele você consegue definir qual o público alvo. E a partir do público alvo, você define qual seria o melhor lugar para ocorrer o evento e em qual dia/horário. 

- [ ] **Quais atividades quero que ocorram no encontro?**
  
  Em geral lembramos mais de palestras, mas nem tudo gira em torno de palestras. Que tal *lightning talks*, *tutorias*, *openspaces*, *pybar*, *dominó* .... ?

- [ ] **Consigo chegar no meu público alvo com as divulgações que irei fazer?**

  As vezes queremos atingir um público, mas não conseguimos chegar neles. Você conseguirá passar essa barreira? Você tem contatinhos para te ajudar nessa divulgação?

--- 

## Definir local e dia
*A maioria dos pontos devem ser preenchidos*

- [ ] **Local foi definido?**
  
  Pense em locais onde o evento ainda não ocorreu ou já tenha ocorrido há muito tempo. Para que assim mais pessoas possam agregar ao PUG e participar dos encontros.
  
- [ ] **O local cabe mais de 30 pessoas?**
  

>   Ou conseguirá comportar o público que você está esperando? Pense bem nisso, a galera não tem frescura senta ate no chão, mas vamos pensar na segurança de todos :smile:

Reviewer: Eu sugeri trocar frescura por besteira porque acho mais ameno e neutro :stuck_out_tongue: 

```suggestion
  Ou conseguirá comportar o público que você está esperando? Pense bem nisso, a galera não tem besteira senta ate no chão, mas vamos pensar na segurança e conforto de todos :smile:
```

Author: O `besteira` saiu como um modo de falar, mas realmente pode ser entendido errado por outras pessoas.

- [ ] **O local permite entrada de pessoas de fora?**
  
  Alguns locais pedem que seja realizado o controle de entrada e saída. Caso isso ocorra com você, lembre-se que isso não é um impedimento, utilize algum sistema de inscrição para efetuar o controle (Exemplo o [Sympla](https://www.sympla.com.br/)) de acesso.
  

>   Sabendo da elevada taxa de ausência no evento quando é preciso realizar inscrições e quando elas são gratuítas. Segue umas dicas de como sobreviver a isso: 

Reviewer: ```suggestion
  Sabendo da elevada taxa de ausência no evento quando é preciso realizar inscrições e quando elas são gratuítas. Segue umas dicas de como contornar isso: 
```

  
  - Forneça mais ingressos do que a capacidade do local;
  - Envie um email de confirmação para os inscritos no evento alguns dias antes do evento;
  - Faça uma lista de espera (um forms no Google é o suficiente);
  - Confirma com as pessoas da lista de espera, caso ocorra liberação de vagas;
  - Tenha "confirmado" a capacidade máxima + 5 (aproximadamente). 
  
- [ ] **O local tem disponibilidade para receber o evento?**
  

>   No caso, a disponibilidade desse local está compatível com o período do mês e até mesmo o mês que o evento deverá ocorrer? Verifique isso antes, **não deixe para verificar a disponibilidade em cima da hora**, principalmente porque tentamos organizar encontros mensais.

Reviewer: Achei essa tua alteração bem melhor, vou sugerir só uma pequena mudança, vê o que tu acha:

```suggestion
  No caso, a disponibilidade desse local está compatível com o período do mês e até mesmo o mês que o evento deverá ocorrer? Verifique isso antes. **Não deixe para verificar a disponibilidade em cima da hora**, principalmente porque tentamos organizar encontros mensais.
```

# Passo a passo inicial
> Aqui está descrito o fluxo principal para criação de um encontro do PUG-PE

Se você está aqui e quer organizar um encontro do PUG-PE, mas não sabe por onde começar, relaxa e se liga nesse checklist :stuck_out_tongue_closed_eyes: :star:

## Antes de organizar o evento
*A maioria dos pontos devem ser preenchidos*

- [ ] **Ter vontade de organizar um encontro**

> - [ ] **Ter amiguinhos com vontade de ajudar você nessa jornada (não precisamos fazer tudo sozinhos, ne mesmo?)**

Reviewer: ```suggestion
- [ ] **Ter amiguinhos com vontade de ajudar você nessa jornada (não precisamos fazer tudo sozinhos, não é mesmo?)**
```

  
  Caso você queira companhia nessa jornada, esse ponto pode ser levado em consideração.
  

>   Se você é estudante em alguma Universidade/Faculdade/Escola e está querendo organizar o encontro onde você estuda, recomendamos que você converse com seus amiguinhos. Provavelmente, terá alguém querendo te ajudar em alguma coisa :smile: 

Reviewer: ```suggestion
  Se você é estudante em alguma Universidade/Faculdade/Escola e está querendo organizar o encontro onde você estuda, recomendamos que você converse com seus colegas. Provavelmente, terá alguém disposto a te ajudar em alguma coisa :smile: 
```

>   Se infelizmente isso não ocorrer, relaxa temos pessoas muito legais e dispostas na organização do PUG-PE que podem te ajudar nessa empreitada nova. 

Reviewer: ```suggestion
  Se, infelizmente, isso não ocorrer, relaxa. Temos pessoas muito legais e dispostas na organização do PUG-PE que podem te ajudar nessa nova empreitada. 
```

  
- [ ] **Ter disponibilidade (ou pessoas disponíveis) para organizar o evento**
  
  Fazer divulgação, gerenciar submissão de palestras, montar grade de horario, organizar o evento no dia, resolver certificados, isso demanda tempo e não deve ser subestimado.
  
  Esse ponto está fortemente relacionado com o anterior, se você acha que sua disponibilidade não é suficiente para fazer tudo isso, recomendamos que você tenha mais alguém com você (mais de duas pessoas é desnecessário). 
  

---

## Pondo as ideias no lugar
*Todos os pontos devem ser preenchidos*

- [ ] **Quero fazer um evento mais introdutorio ou avançado?**
  
  Esse ponto é importante, pois a partir dele você consegue definir qual o público alvo. E a partir do público alvo, você define qual seria o melhor lugar para ocorrer o evento e em qual dia/horário. 

- [ ] **Quais atividades quero que ocorram no encontro?**
  
  Em geral lembramos mais de palestras, mas nem tudo gira em torno de palestras. Que tal *lightning talks*, *tutorias*, *openspaces*, *pybar*, *dominó* .... ?

- [ ] **Consigo chegar no meu público alvo com as divulgações que irei fazer?**

>   As vezes queremos atingir um público, mas não conseguimos chegar neles. Você conseguirá passar essa barreira? Você tem contatinhos para te ajudar nessa divulgação?

Reviewer: Não entendi bem essa, parte, tu poderia me explicar? Ou talvez deixar mais claro, não sei 🤷‍♂

Author: Estava querendo falar que a gente (quem organiza) precisa saber previamente onde irá divulgar o evento. Exemplo:

```
Quero organizar o pug no porto digital, pra avisar pra galera das empresas que o pug existe. Mas eu vou ter como fazer divulgação junto a eles? Vou ter como trazer essa galera pros eventos? Os meios que vou fazer divulgação é acessivel pra eles?
```

Deu pra entender agora? .-.

Reviewer: Entendi! Mas não sei se tá claro do jeito que tá escrito, tu acha que vale a pena reescrever esse parágrafo?

--- 

## Definir local e dia
*A maioria dos pontos devem ser preenchidos*

- [ ] **Local foi definido?**
  

>   Pense em locais onde o evento ainda não ocorreu ou já tenha ocorrido a muito tempo. Para que assim mais pessoas possam agregar ao PUG e participar dos encontros.

Reviewer: ```suggestion
  Pense em locais onde o evento ainda não ocorreu ou já tenha ocorrido há muito tempo. Para que assim mais pessoas possam agregar ao PUG e participar dos encontros.
```

  
- [ ] **O local cabe mais de 30 pessoas?**
  
  Ou conseguirá comportar o público que você está esperando? Pense bem nisso, a galera não tem frescura senta ate no chão, mas vamos pensar na segurança de todos :smile:

- [ ] **O local permite entrada de pessoas de fora?**
  

>   Alguns locais pedem que seja realizado o controle de entrada e saída, caso esse local que você escolheu seja esse tipo de local, realize o evento mesmo assim, mas lembre-se de utilizar algum sistema de inscrição para efetuar o controle ([Sympla](https://www.sympla.com.br/)). 

Reviewer: Achei essa parte um pouco estranha, acho que no começo repete muito "local".
Também acho que seria mais interessante em vez de "realize o evento mesmo assim" trocar para algo do tipo "mas isso não é um impedimento para a realização do evento.
Talvez seja interessante sinalizar de forma mais clara que o Sympla é um exemplo de sistema de inscrição 🤷‍♂ 
O que tu achas?

Author: Dei uma mudada nessa parte, adicionei uns pontos a mais que aprendi esse mês kkkk

  
- [ ] **O local tem disponibilidade para receber o evento?**
  

>   No caso, a disponibilidade desse local está compatível com o período do mês e até mesmo o mês que o evento deverá ocorrer? Verifique isso antes, **não deixe para verificar a disponibilidade encima da hora**, principalmente porque tentamos organizar encontros mensais.

Reviewer: ```suggestion
  No caso, a disponibilidade desse local está compatível com o período do mês e até mesmo o mês que o evento deverá ocorrer? Verifique isso antes, **não deixe para verificar a disponibilidade em cima da hora**, principalmente porque tentamos organizar encontros mensais.
```

>   Caso esse local tenha muitas disponibilidade de horário, siga para o tópico abaixo.

Reviewer: ```suggestion
  Caso esse local tenha muitos horários disponíveis, siga para o tópico abaixo.
```

>   Caso esse local tenha muitas disponibilidade de horário, siga para o tópico abaixo.

Reviewer: ```suggestion
  Caso esse local tenha muitos horários disponíveis, siga para o tópico abaixo.
```

- [ ] **Perguntei no grupo do PUG-PE qual o melhor dia para eles?**
  

>   Temos muitas pessoas no grupo do PUG-PE, os membros mais ativos podem ajudar você a escolher o mehor dia e horário para ocorrer o evento. Recomendamos à você abrir uma enquete no Telegram com as opções de dia/horário, o evento é para a comunidade, sendo assim, vamos fazer a comunidade participar das decições também :smile: 

Reviewer: ```suggestion
  Temos muitas pessoas no grupo do PUG-PE, os membros mais ativos podem ajudar você a escolher o melhor dia e horário para a realização do evento. Recomendamos fazer uma enquete com as opções de dia/horário, o evento é para a comunidade, sendo assim, vamos fazer a comunidade participar das decisões também :smile: 
```

  
- [ ] **Tenho minhas data bem definidas?**

>   Agora que você tem a data do evento bem definida (espero que sejá com uma margem de 30 dias, porque você vai precisar), é hora de definir algumas datas importantes:

Reviewer: ```suggestion
  Agora que você tem a data do evento bem definida(sugerimos que defina a data com antecedência, porque você vai precisar), é hora de definir algumas datas importantes:
```

    - [ ] Dia de abertura das submissões de palestras
    - [ ] Dia de encerramento das submissões de palestras
    - [ ] Período de votação de palestras
    - [ ] Dia para confirmar a disponibilidade dos palestrantes aprovados

>     - [ ] Dia para abrir as inscrições

Reviewer: ```suggestion
    - [ ] Dia para abrir as inscrições (Caso necessário)
```

    - [ ] Dia para resolver o coffebreak
    - [ ] Dia do evento o/

----

## Outros pontos importantes
*Algumas dicas gerais*

- Entre em contato com os responsáveis pelas redes sociais/email oficial/adm do grupo... para te ajudar na divulgação do evento.
- Faça divulgação fora dos grupos oficiais/perfis oficiais do PUG-PE, tem muita gente por ai que programa em Python ou quer começar e não sabe da existência do PUG-PE.
- Seja criativo, faça artes legais para divulgar, pense em frases chamativas, tudo isso ajuda na divulgação ~a espalhar a palavra~ do PUG.

> - Não deixe para organizar o evento encima da hora, tenha pelo menos um período de 1 mês para fazer tudo.

Reviewer: ```suggestion
- Não deixe para organizar o evento em cima da hora, recomendamos que tenha pelo menos o período de 1 mês para fazer tudo.
```
